.github/workflows/release_draft_patch.yml

```diff
@@ -3,10 +3,10 @@ on:
   workflow_dispatch:
     inputs:
       releaseVersion:
-        description: "Latest minor release branch version"
+        description: "Latest minor release branch version (do not create a release branch for a patch!)"
```

Comment:
I would rather put this "(do not create a release branch for a patch!)" on the release_branch workflow. If someone is very new, he would only see this message after having triggered the release_branch branch workflow on a patch.
I would also put in the description here the example of the release version to better visualize it: "X.XXX.0" (like you did in the release_draft_minor workflow)